### PR TITLE
chore: move info component to Zapp-Frameworks repo

### PR DESCRIPTION
**Pull request by alashuk-applicaster**
## Description
Move qb-info-component to Zapp-Frameworks repo.
[Monday ticket](https://applicaster.monday.com/boards/1615228456/views/35072856/pulses/4308722726?userId=17198506)

`Please describe PR here`

<details>
  <summary>Platforms Affected</summary>
  
- [ ] Android
- [ ] Android TV
- [ ] Apple TV (TVOS)
- [ ] Fire TV (FireOS)
- [ ] iOS
- [ ] LG TV (WebOS)
- [ ] Samsung TV (Tizen)
- [ ] Web
</details>

## Known or Pending Issues

`fill in this section with potential issues / limitations the reviewer(s) should know about`

## Screenshots

`If there is some kind of ui change please share screenshots`

## Version Info

`Does this update require a certain version of quickbrick, or does it rely on another pr to work?`

### CODE REVIEW

**Review by inckie — APPROVED** [review]

**Comment by inckie** [comment]
THis should be just new pipesv2:// data source provider, then returns description, then we can use existing cell styles to display it and even handle click if needed (to write to support or copy info for example)

File: plugins/quick-brick-info-component/manifests/android_for_quickbrick.json [hunk 1]
```diff
@@ -0,0 +1,170 @@
+{
+  "api": {},
+  "name": "Info component",
```
Comment (ranmeirman) [inline plugins/quick-brick-info-component/manifests/android_for_quickbrick.json:3]: @alashuk-applicaster let's change the name to "Version Info Component" - no need for the QB.

**Review by ranmeirman — CHANGES_REQUESTED** [review]

File: plugins/quick-brick-info-component/README.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# QB Info Component
```
[the remark this answers is not in the document]
Comment (alashuk-applicaster) [reply plugins/quick-brick-info-component/README.md]: done

File: plugins/quick-brick-info-component/manifests/android_for_quickbrick.json (hunk 1 above)
Comment (alashuk-applicaster) [reply plugins/quick-brick-info-component/manifests/android_for_quickbrick.json]: done

File: plugins/quick-brick-info-component/manifests/ios_for_quickbrick.json [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+  "general": {
+    "fields": []
+  },
+  "rules": {
+    "fields": [
+      {
+        "type": "switch",
+        "label": "Display Version Number",
+        "key": "display_version_number",
+        "initial_value": false
```
Comment (ranmeirman) [inline plugins/quick-brick-info-component/manifests/ios_for_quickbrick.json:45]: why initial value is false? if the component is added, why do you need this flag?

**Review by ranmeirman — CHANGES_REQUESTED** [review]

**Comment by inckie** [comment]
We make this plugin a data source and just use any cell style instead,

**Pull request closed without merging**